server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ExpressionAnalyzer.java

```diff
@@ -268,6 +270,16 @@ public static List<Expression> concatExpressionWithSuffixPaths(
       for (Expression suffixExpression : expression.getExpressions()) {
         extendedExpressions.add(
             concatExpressionWithSuffixPaths(suffixExpression, prefixPaths, patternTree));
+
+        // We just process first input Expression of AggregationFunction,
```

Comment:
may not be true for other agg functions in future, in the comments you need to mention it. or you should do the check